client/src/components/MenuItems.jsx

```diff
@@ -33,21 +34,22 @@ class App extends React.Component {
     fetch(`http://localhost:3000/api/menu/${id}`)
```

Comment:
change this to relative path.

Reply:
It was throwing an error in my test without it. Will change it.